core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -97,7 +97,7 @@
  * @param <ReqT> type of message sent one or more times to the server.
  * @param <RespT> type of message received one or more times from the server.
  */
-@DoNotMock("Use InProcessTransport and make a fake server instead")
+@DoNotMock("Use InProcessServerBuilder and make a full featured in process server instead")
```

Comment:
The "fake server" was not referring to in-process transport. It was referring to the code service code registered via `builder.addService(new ServiceImplBase{} /* this part here */)`. That's why I said "fake service" was being more precise.